openlibrary/plugins/upstream/utils.py

```diff
@@ -106,6 +106,38 @@ def render_template(name, *a, **kw):
         name = name.rsplit(".", 1)[0]
     return render[name](*a, **kw)
 
+
+def kebab_case(upperCamelCase):
+    parts = re.findall(r'[A-Z][^A-Z]*', upperCamelCase)
+    return '-'.join(parts).lower()
+
+
+
+@public
+def render_component(name, attrs=None, json_encode=True):
```

Comment:
@cdrini mentions `attrs` may have an issue (doesn't work)

Reply:
Add comment to README